scopesim/commands/user_commands.py

```diff
@@ -99,15 +99,15 @@ class UserCommands:
     and ``yamls`` must be specified, otherwise scopesim will not know
     where to look for yaml files (only relevant if reading in yaml files)::
 
-        >>> from scopesim.server.database import download_package
-        >>> from scopesim.commands import UserCommands
-        >>>
-        >>> download_package("test_package")
-        >>> cmd = UserCommands(packages=["test_package"],
-        ...                    yamls=["test_telescope.yaml",
-        ...                           {"alias": "ATMO",
-        ...                            "properties": {"pwv": 9001}}],
-        ...                    properties={"!ATMO.pwv": 8999})
+    >>> from scopesim.server.database import download_package
+    >>> from scopesim.commands import UserCommands
+    >>>
+    >>> download_package("test_package")
+    >>> cmd = UserCommands(packages=["test_package"],
+    >>>                    yamls=["test_telescope.yaml",
```

Comment:
The `...`  is actually how your python prompt would look if you copy and paste a multiline command. E.g.
```
print(1,
      2,
      3)
```
looks like this on the command line:
```
>>> print(1,
...       2,
...       3)
1 2 3
>>> 
```
Nevertheless, I'm not sure whether including `>>>`  (and thus `...` as well) is an appropriate way to write docstrings.

I believe that pytest can also execute (and test) docstrings. That is, it can execute the examples in a docstring and then compare the output with the output given in those examples. My idea was to figure out how to make pytest do that, and then ensure the docstrings are written in the appropriate style. This pytest-compliant docstring style might actually be the current style; I'm not sure.

Reply:
Apparently it is `sys.ps1` and `sys.ps2`:
```
>>> import sys
>>> sys.ps1
'>>> '
>>> sys.ps2
'... '
>>> 
```
There apparently is no `sys.ps3`  and `sys.ps4` (but I also do not remember what those do in e.g. bash).

Reply:
Yeah, I tried following [the Numpy style guide](https://numpydoc.readthedocs.io/en/latest/format.html#examples) (thus de-indenting the examples), but didn't read it properly. I'll add the `...` back in. What you're referencing is [doctest](https://docs.python.org/3/library/doctest.html), which can be [integrated in pytest](https://docs.pytest.org/en/7.1.x/how-to/doctest.html), although I've never done that so far. But probably a good idea to look into that 👍